### PR TITLE
added channel and shipmentref

### PR DESCRIPTION
**Pull request by florian-bartels**

### CODE REVIEW

File: internal/mobile_android-picking-v1-pickjob.json [hunk 1]
```diff
@@ -59,6 +75,8 @@
   "required": [
     "id",
     "facilityRef",
+    "shipmentRef",
```
Comment (schoeda) [inline internal/mobile_android-picking-v1-pickjob.json:78]: shipmentRef is nullable ;)

Comment (MarkusWallnerFul) [inline internal/mobile_android-picking-v1-pickjob.json:78]: shipmentRef muss optional sein, da es im collect case kein shipment gibt

**Review by MarkusWallnerFul — CHANGES_REQUESTED** [review]

**Review by schoeda — CHANGES_REQUESTED** [review]

File: internal/mobile_android-picking-v1-pickjob.json (hunk 1 above)
Comment (florian-bartels) [reply internal/mobile_android-picking-v1-pickjob.json:78]: now i am maximal confused ... welchen sinn macht dann der channel ??

Comment (florian-bartels) [reply internal/mobile_android-picking-v1-pickjob.json:78]: ich zitiere mal die stoy
We need shipmentId as non nullable shipmentRes in every PickJob document in mobile_android-picking-v1-pickjobs collection.

Also add the deliverychannel

**Pull request merged**